lib/Tooling/Refactor/RenameIndexedFile.cpp

```diff
@@ -81,7 +81,9 @@ static MatchKind checkOccurrence(const IndexedOccurrence &Occurrence,
   if (BeginLoc.isInvalid())
     return MatchKind::None;
   StringRef SymbolNameStart = Symbol.Name[0];
-  SourceLocation EndLoc = BeginLoc.getLocWithOffset(SymbolNameStart.size());
+  SourceLocation EndLoc = BeginLoc.getLocWithOffset(std::max(
+      SymbolNameStart.size(),
+      size_t(1))); // Take empty Objective-C selector pieces into account.
   if (!SM.isBeforeInTranslationUnit(BeginLoc, EndLoc)) {
```

Comment:
I guess what I was asking is: won't it always be after, since the offset >= 1 now instead of >= 0?